diskutil.py

```diff
@@ -283,6 +298,9 @@ def getDiskDeviceSize(dev):
         return int(__readOneLineFile__("/sys/block/%s/device/block/size" % dev))
     elif os.path.exists("/sys/block/%s/size" % dev):
         return int(__readOneLineFile__("/sys/block/%s/size" % dev))
+    else:
```

Comment:
A `return 0` was added at the end of the function. Do we still need that exception?

Reply:
Right - removed from xenserver#70, will repush without that commit as well

Reply:
done